### ISSUE TITLE
Adopt "Standard" format?

### ISSUE THREAD

**Issue by benthemonkey** [sasstools/sass-lint#417]
Today I noticed that `npm` has now adopted `standard`. The list of packages which use the `standard` format for linting [is growing quite large](https://github.com/feross/standard-packages/blob/master/standard.json).

Pros:
- saves time commenting on PRs, as some of our styles aren't covered by eslint (see [this PR thread](https://github.com/sasstools/sass-lint/pull/327#discussion_r45092785))
- might make it easier for others to contribute
- includes an auto-formatter for many rules (but so does eslint)

Cons:
- clobbers the revision history of files (can't easily use "blame")
- takes time (but I volunteer to perform the migration)
- change is bad :tongue: 

I'm pretty sure debating whether or not to adopt `standard` counts as bikeshedding, which is what `standard` is trying to avoid.

**Comment by DanPurdy** [comment]
I wouldn't consider this until v2.0.0 if we're going to make the switch id rather it didn't all happen in one fell swoop unless we all went forward with it. Absolutely 100% no to one person performing the migration completely.

I personally don't really like the style (it feels like a halfway house between Js and coffeescript to me) but I can see what they're going for and would be open to its adoption if that's the consensus. 

All of the pros you mention we could already handle or are just speculative imo.

TLDR: if we adopt it must be in v2.0 and overall im not sure but willing.

**Comment by Snugug** [comment]
The beauty of ESLint is that, the rules that aren't covered, we can cover.

Personally, as _a linting project_, I'd be opposed to using something whose goal is to deprecate configurable linters in favor of one style to rule them all. It's also got about 1/10 the usage of ESLint.

**Comment by benthemonkey** [comment]
:+1: No problem! That's a very good point @Snugug. Closing.
